### ISSUE TITLE
Not an Issue persay but rather weird

### ISSUE THREAD

**Issue by NhlalukoG** [Mange/rtl8192eu-linux-driver#59]
the driver 8192eu state that monitor mode and other custom modes are not supported except for Managed,Master and Ad-Hoc. That is entirely false. On the RTL8xxxu all those modes work. I'm on an antenna based dongle so RLT8xxxu isnt configured for that(Extremely low signal, we talking -100dBm when the internal says -60dBm).

On os_dep/linux/ioctl_linux.c those modes were intentionally removed and on the rest of the source, for what good reason i dont know. If u modify one of the cases from line 1465 and rename it to monitor instead and use iwconfig **** mode monitor, it does recognise just not able to apply since the mode is missing entirely.

bottom line, the chipset supports all these fancy modes but the source does not.

**Comment by CGarces** [comment]
Monitor mode is not supported?
Based on the [changelog](https://github.com/Mange/rtl8192eu-linux-driver/issues/36) is supported since v4.3.15.2.

In any case I don't have enough skills to add new features to the existing drivers, see the note at [readme.md](https://github.com/Mange/rtl8192eu-linux-driver/blob/realtek-4.4.x/README.md)

The main purpose of this repo is maintain the last source code from Realtek and keep it updated with lasts versions of kernels & GCC. Maybe you should test RTL8xxxu driver and send bug reports to improve the mainstream driver.
